CMakeLists.txt

```diff
@@ -1,47 +1,65 @@
-cmake_minimum_required(VERSION 3.16.3)
+cmake_minimum_required(VERSION 3.0.2)
 project(affordance_primitives)
 
-find_package(ament_cmake REQUIRED)
-ament_package_xml()
-
+add_compile_options(-std=c++14)
 if(NOT "${CMAKE_CXX_STANDARD}")
-  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD 14)
 endif()
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Are both of these lines necessary? I thought CMake perferred set(standard) over add_compile_options due to portability